doc/src/stdlib/collections.md

```diff
@@ -177,17 +178,17 @@ Partially implemented by:
 as the hashing function for the key, and [`isequal()`](@ref) to determine equality. Define these
 two functions for custom types to override how they are stored in a hash table.
 
-`ObjectIdDict` is a special hash table where the keys are always object identities.
+[`ObjectIdDict`](@ref) is a special hash table where the keys are always object identities.
 
-`WeakKeyDict` is a hash table implementation where the keys are weak references to objects, and
+[`WeakKeyDict`](@ref) is a hash table implementation where the keys are weak references to objects, and
 thus may be garbage collected even when referenced in a hash table.
 
 [`Dict`](@ref)s can be created by passing pair objects constructed with `=>()` to a [`Dict`](@ref)
 constructor: `Dict("A"=>1, "B"=>2)`. This call will attempt to infer type information from the
 keys and values (i.e. this example creates a `Dict{String, Int64}`). To explicitly specify types
 use the syntax `Dict{KeyType,ValueType}(...)`. For example, `Dict{String,Int32}("A"=>1, "B"=>2)`.
 
-[`Dict`](@ref)s may also be created with generators. For example, `Dict(i => f(i) for i = 1:10)`.
+[`Dict`](@ref) may also be created with generators. For example, `Dict(i => f(i) for i = 1:10)`.
```

Comment:
Sorry, that slipped.  I think, after #19534 it should read: "Associative collections may also..."